examples/demo_threading/qml/SensorUI.qml

```diff
@@ -48,7 +48,7 @@ Item {
         anchors.margins: -15
 
         Image {
-            source: "./images/sensorefect.png"
+            source: "/images/sensorefect.png"
```

Comment:
seems fun that in the build.rs `                "../images/sensorefect.png",` is specified but here `"/images/sensorefect.png"` is specified :thinking:

Reply:
And does `"images/sensorefect.png"` also work ?

Reply:
Yep, somehow it Just Works. I guess rcc knows to ignore `..` :shrug:

Reply:
> And does "images/sensorefect.png" also work ?

No, not here. But somehow leaving out the `/` is required for the paths to the QML pages in examples/qml_features/qml/main.qml.

Reply:
What does the generated qrc file look for this ?

Reply:
```
❯ cat build/cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx-qt-demo-threading-c3d6fe097eb83322/out/qml_modules/com/kdab/energy/qml_module_resources.qrc
<RCC>
<qresource prefix="/">
    <file alias="/qt/qml/com/kdab/energy">/home/be/sw/cxx-qt/build/./cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx-qt-demo-threading-c3d6fe097eb83322/out/qml_modules/com/kdab/energy</file>
</qresource>
<qresource prefix="/qt/qml/com/kdab/energy">
    <file alias="../qml/Button.qml">/home/be/sw/cxx-qt/examples/demo_threading/qml/Button.qml</file>
    <file alias="../qml/MainWindow.qml">/home/be/sw/cxx-qt/examples/demo_threading/qml/MainWindow.qml</file>
    <file alias="../qml/Panel.qml">/home/be/sw/cxx-qt/examples/demo_threading/qml/Panel.qml</file>
    <file alias="../qml/SensorUI.qml">/home/be/sw/cxx-qt/examples/demo_threading/qml/SensorUI.qml</file>
    <file alias="../qml/SideText.qml">/home/be/sw/cxx-qt/examples/demo_threading/qml/SideText.qml</file>
    <file alias="../images/activeInner.png">/home/be/sw/cxx-qt/examples/demo_threading/images/activeInner.png</file>
    <file alias="../images/activeOuter.png">/home/be/sw/cxx-qt/examples/demo_threading/images/activeOuter.png</file>
    <file alias="../images/beach1.png">/home/be/sw/cxx-qt/examples/demo_threading/images/beach1.png</file>
    <file alias="../images/beach2.png">/home/be/sw/cxx-qt/examples/demo_threading/images/beach2.png</file>
    <file alias="../images/bg.png">/home/be/sw/cxx-qt/examples/demo_threading/images/bg.png</file>
    <file alias="../images/iconSensors.png">/home/be/sw/cxx-qt/examples/demo_threading/images/iconSensors.png</file>
    <file alias="../images/iconwirless.png">/home/be/sw/cxx-qt/examples/demo_threading/images/iconwirless.png</file>
    <file alias="../images/inactiveInner.png">/home/be/sw/cxx-qt/examples/demo_threading/images/inactiveInner.png</file>
    <file alias="../images/inactiveOuter.png">/home/be/sw/cxx-qt/examples/demo_threading/images/inactiveOuter.png</file>
    <file alias="../images/kdabLogo.png">/home/be/sw/cxx-qt/examples/demo_threading/images/kdabLogo.png</file>
    <file alias="../images/level0.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level0.png</file>
    <file alias="../images/level0i.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level0i.png</file>
    <file alias="../images/level1.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level1.png</file>
    <file alias="../images/level1i.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level1i.png</file>
    <file alias="../images/level2.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level2.png</file>
    <file alias="../images/level2i.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level2i.png</file>
    <file alias="../images/level3.png">/home/be/sw/cxx-qt/examples/demo_threading/images/level3.png</file>
    <file alias="../images/ocean.png">/home/be/sw/cxx-qt/examples/demo_threading/images/ocean.png</file>
    <file alias="../images/panel.png">/home/be/sw/cxx-qt/examples/demo_threading/images/panel.png</file>
    <file alias="../images/qt-logo.png">/home/be/sw/cxx-qt/examples/demo_threading/images/qt-logo.png</file>
    <file alias="../images/RLogo.png">/home/be/sw/cxx-qt/examples/demo_threading/images/RLogo.png</file>
    <file alias="../images/RLogolarge.png">/home/be/sw/cxx-qt/examples/demo_threading/images/RLogolarge.png</file>
    <file alias="../images/rust-logo-white.png">/home/be/sw/cxx-qt/examples/demo_threading/images/rust-logo-white.png</file>
    <file alias="../images/sensor.png">/home/be/sw/cxx-qt/examples/demo_threading/images/sensor.png</file>
    <file alias="../images/sensorefect.png">/home/be/sw/cxx-qt/examples/demo_threading/images/sensorefect.png</file>
    <file alias="../images/sideshadow.png">/home/be/sw/cxx-qt/examples/demo_threading/images/sideshadow.png</file>

    <file alias="qmldir">/home/be/sw/cxx-qt/build/./cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx-qt-demo-threading-c3d6fe097eb83322/out/qml_modules/com/kdab/energy/qmldir</file>
</qresource>
</RCC>
```

Reply:
Nice, our examples create some fun because the `qml/` folder is "above" us, but typical Rust apps (like the cargo_without_cmake) example has the qml beside the build.rs and creates much more sane paths :-)